python/graphstorm/eval/evaluator.py

```diff
@@ -1034,64 +1141,21 @@ def _get_major_score(self, score):
             major_score = score[self.major_etype]
         return major_score
 
-    def evaluate(self, val_rankings, test_rankings, total_iters):
-        """ `GSgnnLinkPredictionTrainer` and `GSgnnLinkPredictionInferrer` will call this function
-        to compute validation and test mrr scores.
+    def get_val_score_rank(self, val_score):
+        """ Get the rank of the given ``major_etype`` validation score by comparing its value
+        to the existing value list. If use the default ``major_etype``, will use the summation
```

Comment:
```suggestion
        to the existing value list. If using the default ``major_etype``, the rank will be the summation
```

Reply:
Follow Xiang's suggestion, revised this sentence to clarify `major_etyp`.